llm_bench/python/who_what_benchmark/examples/openvino_cb.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import PosixPath
+import os
+
+import whowhatbench
+from whowhatbench.wwb import load_dataset
+from optimum.intel.openvino import OVModelForCausalLM
+
+from openvino_genai import ContinuousBatchingPipeline, SchedulerConfig, GenerationConfig, CacheEvictionConfig, AggregationMode
+
+from openvino_tokenizers import convert_tokenizer
+from openvino import serialize
+from transformers import AutoTokenizer
+
+model_id = "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
+MAX_NEW_TOKENS = 128
+SEQS_PER_REQUEST = 5
+MAX_SEQUENCES = 100
+
+
+model = OVModelForCausalLM.from_pretrained(model_id, export=True, trust_remote_code=True)
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+model_path = PosixPath("/tmp/models_wwb") / model_id
```

Comment:
Tmp directory should be created with `tempfile` package I guess.

Reply:
Done